lib/data/dataset.js

```diff
@@ -68,7 +68,7 @@ const getDataset = (xml) => {
       return Option.none();
     if (version.isEmpty())
       throw Problem.user.invalidEntityForm({ reason: 'Entities specification version is missing.' });
-    else if (!version.get().startsWith('2022.1.'))
+    else if (!(version.get().startsWith('2022.1.') || version.get().startsWith('2023.1.')))
```

Comment:
You could use the `semver` package if you wanted. It's already a dependency. (Though it's a dev dependency, so you'd probably need to reinstall it as a prod dependency.) This also looks fine though.

Reply:
maybe *next* version bump?